ggml.c

```diff
@@ -12620,6 +12650,46 @@ static void ggml_compute_forward_clamp(
 
 // ggml_compute_forward_rope
 
+static inline float rope_yarn_ramp(const float low, const float high, const int i0) {
```

Comment:
the only thing `inline` does here is suppressing the warning, if the function is unused.
not sure that is desired...